lib/r10k/api.rb

```diff
@@ -25,6 +28,9 @@ module R10K
   module API
     extend R10K::Logging
 
+    class UnresolvableError < StandardError; end
```

Comment:
this needs to be fleshed out a bit to support the way I'm actually using it below